MedicalDocManagement.BLL/Services/ChildCardsService.cs

```diff
@@ -139,19 +143,38 @@ public ParentChildCardDTO AddParentIntoChildCard(ParentChildCardDTO parentChildC
 
             return parentChildCardDTO;
         }
-            
-        public List<ChildCardDTO> FindChildCards(ChildCardDTO childCardDTO)
+
+        public List<ChildCardDTO> FindFirst20ChildCards(IViewPatientData viewPatientData)
```

Comment:
Feature doesn't contain case to return exactly 20 patients.
Return full list of patients that store in database.